cdap-ui/app/cdap/styles/common.scss

```diff
@@ -43,9 +43,9 @@ body {
   font-family: var(--font-family);
 
   #app-container {
-    margin-top: 50px;
+    margin-top: 48px;
```

Comment:
Same as the other comment: https://github.com/cdapio/cdap/pull/11020/files#diff-7aed8587c8ae4dfd5468897ea990f6baR46 already done in the style PR. Can you revert this change so that we can pick it up from there?